Remotion/ObjectBinding/Validation.UnitTests/PropertyValidatorToBusinessObjectPropertyConstraintConverterTest.cs

```diff
@@ -15,17 +15,82 @@
 // along with re-motion; if not, see http://www.gnu.org/licenses.
 // 
 using System;
+using System.Linq;
 using NUnit.Framework;
+using Remotion.ObjectBinding.BusinessObjectPropertyConstraints;
+using Remotion.Validation.Implementation;
+using Remotion.Validation.Validators;
 
 namespace Remotion.ObjectBinding.Validation.UnitTests
 {
   [TestFixture]
   public class PropertyValidatorToBusinessObjectPropertyConstraintConverterTest
   {
     [Test]
-    [Ignore("TODO RM-5906")]
-    public void Convert ()
+    public void Convert_WithDifferentMaxLengthValidators_ContainsLowerMaxLength ()
     {
+      var validators = new []
+                       {
+                         new MaximumLengthValidator(7, new InvariantValidationMessage("Max length is 7")),
+                         new MaximumLengthValidator(5, new InvariantValidationMessage("Max length is 5"))
```

Comment:
```suggestion
                         new MaximumLengthValidator(7, new InvariantValidationMessage("Max length is 7")),
                         new MaximumLengthValidator(5, new InvariantValidationMessage("Max length is 5")),
                         new MaximumLengthValidator(7, new InvariantValidationMessage("Max length is 6")),
```

For cases such as this, I like to put the expected value in the middle to prevent code changes that simply fall back to the first / last entry when selecting the value. 

A similiar scenario is when I look for multiple values that also have a skip-condition. There, I make sure the skip-condition applies to a middle item and then assert that the first and last item are part of the result. That way, I make sure the skip condition results in a skip and not a loop-break.